scvi/data/anndata/manager.py

```diff
@@ -99,8 +101,8 @@ def register_fields(
         ----------
         adata
             AnnData object to be registered.
-        source_setup_dict
-            Setup dictionary created after registering an AnnData using an AnnDataManager object.
+        source_registry
+            Registry created after registering an AnnData using an AnnDataManager object.
```

Comment:
```suggestion
            Registry created after registering an AnnData using an :class:`scvi.data.AnnDataManager` object.
```
not 100% sure this is correct, but syntax is